qutebrowser/utils/utils.py

```diff
@@ -42,6 +42,11 @@
 from PyQt5.QtCore import QUrl, QVersionNumber
 from PyQt5.QtGui import QClipboard, QDesktopServices
 from PyQt5.QtWidgets import QApplication
+# We cannot use the stdlib version on 3.7-3.8 because we need the files() API.
+if sys.version_info >= (3, 9):
+    import importlib.resources as importlib_resources
+else:
+    import importlib_resources
```

Comment:
Feel free to ignore that one, I'll fix things up when regenerating the `requirements.txt`.